components/button/stories/button.test.js

```diff
@@ -109,8 +109,6 @@ export const ButtonGroups = Variants({
 			customStyles: {
 				"max-inline-size": "480px",
 			},
-			iconName: "Edit",
-			iconSet: "workflow",
```

Comment:
These are already defined in the template being used, so it should be safe to remove them.

Reply:
Hm, hopefully I'm not misunderstanding the test template, but I'm pretty sure they're already defined. The "Line wrap" test scenario calls for the `ButtonIconGroup` template. `ButtonIconGroup` has 4 wrapping buttons- 2 with workflow icons and 2 with UI icons. So I think we have the workflow examples & spacing covered. These extra `iconName` and `iconSet` args were just repeats and not doing anything anyways (since the icons are defined in `ButtonIconGroup` instead).

If that's not accurate, I can definitely revert this!